lib/server.js

```diff
@@ -78,19 +77,7 @@ async function initConfiguration(rawConfig) {
     config.repoRoot = path.resolve(config.repoRoot);
     await fse.ensureDir(config.repoRoot);
 
-    // configure logger
```

Comment:
Please see previous comment in `config.js`. Processing the configuration (providing defaults, creating folders etc) should happen here.